testng/src/main/java/io/specto/hoverfly/testng/HoverflyExtension.java

```diff
@@ -0,0 +1,484 @@
+/**
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this classpath except in compliance with
+ * the License. You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ * <p>
+ * Copyright 2016-2016 SpectoLabs Ltd.
+ */
+package io.specto.hoverfly.testng;
+
+import io.specto.hoverfly.junit.core.*;
+import io.specto.hoverfly.junit.core.model.Simulation;
+import io.specto.hoverfly.junit.dsl.RequestMatcherBuilder;
+import io.specto.hoverfly.junit.dsl.StubServiceBuilder;
+import io.specto.hoverfly.junit.verification.VerificationCriteria;
+import org.apache.commons.lang3.StringUtils;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.*;
+import java.util.function.Predicate;
+
+import static io.specto.hoverfly.junit.core.HoverflyConfig.localConfigs;
+import static io.specto.hoverfly.junit.core.HoverflyConstants.DEFAULT_HOVERFLY_EXPORT_PATH;
+import static io.specto.hoverfly.junit.core.HoverflyMode.*;
+import static io.specto.hoverfly.junit.core.SimulationSource.file;
+import static io.specto.hoverfly.testng.HoverflyExtensionUtils.*;
+
+public class HoverflyExtension {
```

Comment:
The first time I looked at it I thought it's a copy of the Junit 5 extension 😆  It'd be good to make the name more idiomatic for the TestNg world.

Reply:
Changed to HoverflyTestNG